salesagility/SuiteCRM
# Convert line endings

## Description



Fixes CLRF line endings

## How To Test This

1. Pull down the branch.
2. Run "file LICENSE-MIT" in terminal and check the line ending is correct.

## Types of changes


### Final checklist
